src/commands/remember/rememberRangeGrab.js

```diff
@@ -1,5 +1,5 @@
 const { ApplicationCommandOptionType, PermissionFlagsBits } = require('discord.js');
-const remeberMessagesMethods = require("../../utils/rememberMessages");
+const remeberMessagesUtils = require("../../utils/rememberMessages");
```

Comment:
We should fix this spelling; I'm counting 14 instances of "remeber" in this PR.

Reply:
These have been fixed in [this PR](https://github.com/RIT-Crowd-Comic/Bot/pull/14). Hoping doing it like this decreases the chances of merge conflicts